test/npm/test-npm-test.js

```diff
@@ -135,6 +143,28 @@ test('npm-test: timeout', (t) => {
   });
 });
 
+test('npm-test: tmpdir is redirected', (t) => {
+  const context = makeContext.npmContext(
+    'omg-i-write-to-tmpdir',
+    packageManagers,
+    sandbox,
+    {
+      npmLevel: 'silly'
+    }
+  );
+  context.npmConfigTmp = writeTmpdirTemp;
+  packageManagerTest('npm', context, (err) => {
+    t.error(err);
+    t.ok(
+      fs.existsSync(
+        path.join(writeTmpdirTemp, 'omg-i-write-to-tmpdir-testfile')
+      ),
```

Comment:
Should this clean up the test file afterwards?

Reply:
@addaleax the `npm-test: teardown` at the end of this file wipes the entire `sandbox` used by the tests in this file.

Reply:
Right, thank you!